tutorials/tree/dictionary/data2Tree.cpp

```diff
@@ -0,0 +1,19 @@
+// ##########################################################################
+// # Alvaro Tolosa Delgado @ IFIC(Valencia,Spain)  alvaro.tolosa@ific.uv.es #
+// # Copyright (c) 2018 Alvaro Tolosa. All rights reserved.		 #
+// ##########################################################################
+
+// This file contains the definition of the class method
+// The class method was declared in the corresponding .hpp file
+
+#include "data2Tree.hpp"
+#include <iostream>
+
+void myDetectorData::clear()
+{
+   time = 0;
+   energy = 0;
+   detectorID = 0;
+   correlatedDetectors_v.clear();
+   return;
```

Comment:
C/C++ functions are guaranteed to return anyway when execution reaches the end of their body.
```suggestion
```